dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/basic/warlibs/webinfLibJar/beans.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
```

Comment:
is this intentionally unversioned?

Reply:
I don't think so, though it is consistent with the other test archives for the warlibs test.

Should probably update all of them to have a version.

Reply:
Versions added to all beans.xmls